app/code/Magento/Checkout/Test/Mftf/Test/StorefrontGuestCheckoutTest/StorefrontCheckoutSummaryItemsInCartLabelPluralized.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="StorefrontCheckoutSummaryItemsInCartLabelPluralized">
```

Comment:
Hi @saphaljha awesome work! Thank you. One minor thing, please, add the "...Test" suffix to the test name and the file name correspondingly
```suggestion
    <test name="StorefrontCheckoutSummaryItemsInCartLabelPluralizedTest">
```

Reply:
@rogyar , I have update my commit as per your suggestions, thank you for review.